monitor/event.go

```diff
@@ -10,14 +10,15 @@ import (
 type EventName string
 
 const (
-	UpdateSidechainAddr   EventName = "UpdateSidechainAddr"
-	ConfirmParamProposal  EventName = "ConfirmParamProposal"
-	Delegate              EventName = "Delegate"
-	CandidateUnbonded     EventName = "CandidateUnbonded"
-	ValidatorChange       EventName = "ValidatorChange"
-	IntendWithdrawSgn     EventName = "IntendWithdraw"
-	IntendWithdrawChannel EventName = "IntendWithdrawChannel"
-	IntendSettle          EventName = "IntendSettle"
+	UpdateSidechainAddr          EventName = "UpdateSidechainAddr"
+	ConfirmParamProposal         EventName = "ConfirmParamProposal"
+	Delegate                     EventName = "Delegate"
+	CandidateUnbonded            EventName = "CandidateUnbonded"
+	ValidatorChange              EventName = "ValidatorChange"
+	IntendWithdrawSgn            EventName = "IntendWithdraw"
+	IntendWithdrawChannel        EventName = "IntendWithdraw"
+	IntendWithdrawChannelWrapped EventName = "IntendWithdrawChannel"
```

Comment:
Suggest to rename:
```
IntendWithdraw        EventName = "IntendWithdraw"
IntendWithdrawSgn     EventName = "IntendWithdrawSgn"
IntendWithdrawChannel EventName = "IntendWithdrawChannel"
```
Then monitor on `IntendWithdraw` for both SGN and Channel, switch on `IntendWithdrawSgn` and `IntendWithdrawChannel` when parse event

Reply:
Done.